src/features/App/WorkspaceManager/ProfileCreator/index.tsx

```diff
@@ -192,14 +208,14 @@ export const ProfileCreator: FC<ProfileCreatorProps> = ({
 
 					{profileNameError && <Text color="red.500">{profileNameError}</Text>}
 				</VStack>
-
 				<VStack w="100%" alignItems="start">
+					<Text>Password</Text>
 					<Input
 						ref={passwordInputRef}
 						variant="filled"
 						size="lg"
 						type="password"
-						placeholder="Enter password"
+						placeholder="MyPa$$word"
```

Comment:
Add `e.g.` here too. I'm not sure that's right way it's just seems more clear for me, so you could research the best practice how to write placeholders and why.